bambi/backend/utils.py

```diff
@@ -5,7 +5,16 @@
 import pytensor.tensor as pt
 import pymc as pm
 
-MAPPING = {"Cumulative": pm.Categorical, "StoppingRatio": pm.Categorical}
+
+def horseshoe(name, tau_nu=3, lam_nu=1, dims=None):
+    tau = pm.HalfStudentT(f"{name}_tau", nu=tau_nu)
+    lam = pm.HalfStudentT(f"{name}_lam", nu=lam_nu, dims=dims)
+    beta_raw = pm.Normal(f"{name}_raw", 0, 1, dims=dims)
+    beta = pm.Deterministic(name, beta_raw * tau**2 * lam**2, dims=dims)
+    return beta
```

Comment:
Could you add a docstring here explaining what the arguments are? 

* `name` is the name of the parameters as registered in the PyMC model
* `tau_nu` the degrees of freedom of `tau`
* `lam_nu` the degrees of freedom of `lam`
* `dims` dimensions passed to PyMC. 

Also, 

* Mention in the docstring this is an internal function which is not supposed to be used by users. 
* Include that by default the prior on `lam` is equivalent to a `HalfCauchy` (because of `lam_nu=1`)

Reply:
@julianlheureux you can have a look at some examples of docstrings here 

https://github.com/bambinos/bambi/blob/038610c306007484c259e24ddbd6461669868066/bambi/transformations.py

and this is just a particular example

https://github.com/bambinos/bambi/blob/038610c306007484c259e24ddbd6461669868066/bambi/transformations.py#L152-L166